site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -107,12 +95,3 @@ When searching for users by application or any fields on an application, it is n
 The following screenshot shows an Elasticsearch JSON query being constructed to search for users that match the email pattern `*@fusionauth.io`, are registered to the `Pied Piper` application, and are assigned the `admin` role:
 
 image::user-search-json-query.png[User Search by JSON Query,width=1200,role=shadowed top-cropped]
```

Comment:
Would put another link:

To learn more about the Elasticsearch search engine in general, view the link:...[search core concepts section].